### PR TITLE
docs(docker): add section for overriding hard-coded image name

### PR DESCRIPTION
**Pull request by developer-guy**
Co-authored-by: Furkan <furkan.turkal@trendyol.com>

Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

Add another section to the Docker's page to tell people how they can override registry and image names within `.goreleaser.yml` file. 

To give information on how they can overcome this issue

...

### CODE REVIEW

File: www/docs/customization/docker.md [hunk 1]
```diff
@@ -279,3 +279,52 @@ dockers:
     build_flag_templates:
     - "--builder=heroku/buildpacks:20"
 ```
+
+## How to override hardcoded registry and image name ?
```
Comment (caarlos0) [inline www/docs/customization/docker.md:283]: not so sure if this should be here or in the cookbooks section... thoughts?

Comment (developer-guy) [reply www/docs/customization/docker.md:283]: yeah, if you think that cookbooks page is the right place to put this, then we can move it over there.

**Comment by caarlos0** [comment]
lets move it to the cookbooks section @developer-guy ?

**Comment by developer-guy** [comment]
> lets move it to the cookbooks section @developer-guy ?

I already moved it to there 👌

**Comment by caarlos0** [comment]
> I already moved it to there 👌

oh, my bad, I meant, in a new file [here](https://github.com/goreleaser/goreleaser/tree/main/www/docs/cookbooks)

**Comment by developer-guy** [comment]
ah okay, done. 😋

**Pull request merged**